setup.py

```diff
@@ -1,20 +1,31 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+
 from setuptools import setup
+from testr.setup_helper import cmdclass
```

Comment:
I thought we were just going to cut this, but I don't have skin in the game.

Reply:
I think we said it might be a good thing in general, but that change is not needed for this PR so we can revisit.